examples/thread_safety/readme.md

```diff
@@ -0,0 +1,17 @@
+### Run
+
+```sh
+v -prod -autofree ./queue.v -o ./queue.c && \
+gcc ./queue.c -o ./queue.out && \
+./queue.out
```

Comment:
There is no need for that complexity.

`v -prod -cc gcc ./queue.v` is enough to produce a `queue` executable, compiled with gcc.

Reply:
If you use gcc manually you will also have to add `-lpthread` depending on the platform etc (it is needed on Ubuntu for example).

Using just V, it can take care of that for you.